dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -70,9 +64,9 @@ export const NoExpiringRuns = () => {
       <TextContent className="no-runs-wrapper">
         <Text component={TextVariants.h3}> You have no runs expiring soon</Text>
         <Text component={TextVariants.p}>
-          Runs that have expiration date within next {EXPIRATION_DAYS_LIMIT}{" "}
-          days will appear here. These runs will be automatically removed from{" "}
-          the system if left unacknowledged.
+          Runs that have expiration date within next{" "}
+          {CONSTANTS.EXPIRATION_DAYS_LIMIT} days will appear here. These runs
+          will be automatically removed from the system if left unacknowledged.
```

Comment:
Updated! The user can extend the expiration date by updating it from the Saved/New runs component

Reply:
I'm a little fuzzy on the details, but the Server configuration imposes a maximum limit, the user can pick a value which is less than that, and then the user can change it up to but not exceeding the Server's maximum...right?

And, the expiration is based on when the dataset is uploaded, not when it is "acknowledged" -- even if it is "saved", it will still be removed when the expiration date is reache, and there's nothing the user can do about that...right?

(@dbutenhof?)